src/background/extension.ts

```diff
@@ -190,19 +195,33 @@ export class Extension {
 
     private startAutoTimeCheck() {
         setInterval(() => {
-            if (!this.ready || this.user.settings.automation !== 'time') {
+            if (this.user.settings.automation !== 'time' && this.user.settings.automation !== 'system') {
```

Comment:
we still need to check by interval as a fallback solution

Reply:
I suppose interval code is not needed

That was a workaround for non-firing event for media query change